apps/web/lib/apps/[slug]/[...pages]/getServerSideProps.ts

```diff
@@ -22,36 +21,23 @@ type AppPageType = {
 
 type Found = {
   notFound: false;
-  Component: AppPageType["default"];
   getServerSideProps: AppPageType["getServerSideProps"];
 };
 
 type NotFound = {
   notFound: true;
 };
 
-// TODO: It is a candidate for apps.*.generated.*
-const AppsRouting = {
-  "routing-forms": RoutingFormsRoutingConfig,
-  typeform: TypeformRoutingConfig,
```

Comment:
TypeForm page does not have getServerSideProps, and hence it would've been caught by the `appPage` validation in line 49. Hence, I remove `AppsRouting` entirely and use `routingServerSidePropsConfig` directly